index.jsx

```diff
@@ -22,10 +23,15 @@ class Variable extends React.Component {
     this.props.changeSelected(event.target.value);
   }
 
+  getName() {
+    return this.props.name || this.props.variable;
+  }
+
   getDefault() {
-    const def = this.props.defaults.filter(Boolean).find(d => d.name === this.props.variable) || {};
+    const def = this.props.defaults.filter(Boolean).find(d => d.name === this.getName()) || {};
+
     if (def.default) return def.default;
-    return this.props.variable.toUpperCase();
+    return this.getName().toUpperCase();
```

Comment:
just incase `getName()` returns `undefined`.

```suggestion
    return this.getName()?.toUpperCase() || undefined;
```